package.json

```diff
@@ -7,6 +7,10 @@
     "test": "tape test/index.js",
     "update-abi-registry": "node --unhandled-rejections=strict scripts/update-abi-registry.js"
   },
+  "files": [
+    "abi_registry.json",
+    "README.md"
```

Comment:
The readme should be implied. See https://docs.npmjs.com/cli/v9/configuring-npm/package-json?v=true#files